roles/ceph-mon/templates/ceph-mon.service.j2

```diff
@@ -23,8 +23,12 @@ ExecStart=/usr/bin/docker run --rm --name ceph-mon-%i --net=host \
    --net=host \
    {% endif -%}
    -e CEPH_DAEMON=MON \
-   -e MON_IP={{ hostvars[inventory_hostname]['ansible_default_' + ip_version]['address'] }} \
+   {% if ip_version == 'ipv4' %}
+   -e MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface][ip_version]['address'] }} \
    -e CEPH_PUBLIC_NETWORK={{ ceph_mon_docker_subnet }} \
+   {% elif ip_version =='ipv6' %}
+   -e MON_IP={{ hostvars[inventory_hostname]['ansible_' + ceph_mon_docker_interface][ip_version][0]['address'] }} \
```

Comment:
I noticed this difference as well, but the facts I was looking at did have an address for IPV6 in ``ansible_default_ipv6``.  Another thing I noticed was the the ``[ansible_$interface][ipv6]`` list had two elements. Are we sure that index 0 is the one we always want?